isis/src/base/objs/CSMCamera/CSMCamera.cpp

```diff
@@ -603,41 +830,85 @@ namespace Isis {
 
 
   /**
-   * Returns the pixel ifov offsets from center of pixel.  For vims this will be a rectangle or
-   * square, depending on the sampling mode.  The first vertex is the top left.
+   * Returns the pixel ifov offsets from center of pixel. The first vertex is the top left.
    *
    * The CSM API does not support this type of internal information about the sensor.
+   *
+   * @returns @b QList<QPointF> The field of view offsets
    */
   QList<QPointF> CSMCamera::PixelIfovOffsets() {
     QString msg = "Pixel Field of View is not supported for CSM camera models";
     throw IException(IException::User, msg, _FILEINFO_);
   }
 
 
+  /**
+   * Get the body fixed position of the sun in kilometers.
+   *
+   * This is not supported for CSM sensors because we cannot get the position
+   * of the sun, only the illumination direction.
+   *
+   * @param[out] p The position of the sun
+   */
   void CSMCamera::sunPosition(double p[3]) const {
     QString msg = "Sun position is not supported for CSM camera models";
     throw IException(IException::Programmer, msg, _FILEINFO_);
   }
 
 
+  /**
+   * Get the SpicePosition object the contains the state information for the sun in J2000.
```

Comment:
Single typo: Potentially update the first "the" to "that".

Reply:
Fixed, thanks for catching that.